### PR TITLE
Story 1.3 olia

### PR DESCRIPTION
**Pull request by Olga04051995**

### CODE REVIEW

File: src/app/caesar-portal/group-list/group-list.component.scss [hunk 1]
```diff
@@ -0,0 +1,17 @@
+.group-list-container {
+  padding: 25px;
+  border: 1px solid #ccc;
+  border-radius: 5px;
+  margin-top: 20px;
```
Comment (ksenya-ta) [inline src/app/caesar-portal/group-list/group-list.component.scss]: it is better to combine margins in one block

File: src/app/caesar-portal/group-list/group-list.component.scss [hunk 2]
```diff
@@ -0,0 +1,17 @@
+.group-list-container {
+  padding: 25px;
+  border: 1px solid #ccc;
```
Comment (ksenya-ta) [inline src/app/caesar-portal/group-list/group-list.component.scss]: move all colors to variables

File: src/app/caesar-portal/group-list/group-list.component.scss [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+  border-radius: 5px;
+  margin-top: 20px;
+  min-height: calc(100% - 100px);
+  margin-right: 20px;
+  margin-left: 20px;
+  .search-btn {
```
Comment (ksenya-ta) [inline src/app/caesar-portal/group-list/group-list.component.scss]: 1 empty line before each new block

File: src/app/caesar-portal/group-list/group-item/group-item.component.scss [hunk 4]
```diff
@@ -0,0 +1,16 @@
+:host{
+    height: 100px;
+    display: inline-block;
+    text-align: center;
+    margin: 10px;
+    background-color: #337ab7;
```
Comment (ksenya-ta) [inline src/app/caesar-portal/group-list/group-item/group-item.component.scss]: move all colors hashes to variables

File: src/app/caesar-portal/group-list/group-item/group-item.component.scss [hunk 5]
```diff
@@ -0,0 +1,16 @@
+:host{
```
Comment (ksenya-ta) [inline src/app/caesar-portal/group-list/group-item/group-item.component.scss]: pls add a space before each curly bracket

File: src/app/caesar-portal/common/models/group.ts [hunk 6]
```diff
@@ -0,0 +1,7 @@
+export class Group {
+  groupId: number;
+  name: string;
+  startDate: string;
+  finishDate: string;
+  experts : string[];
```
Comment (ksenya-ta) [inline src/app/caesar-portal/common/models/group.ts]: remove extra space here

File: src/app/app.component.ts [hunk 7]
```diff
@@ -1,9 +1,27 @@
...
 
 @Component({
   selector: 'app-root',
   templateUrl: './app.component.html',
 })
 
 export class AppComponent {
+  constructor(private http: Http) {
+    //TODO: remove when auth logic is done 
+    const body = JSON.stringify({
```
Comment (ksenya-ta) [inline src/app/app.component.ts]: implement OnInit interface and move this logic to ngOnInit

File: src/app/app.component.ts [hunk 8]
```diff
@@ -1,9 +1,27 @@
...
+      username: "OlegShvets",
```
Comment (ksenya-ta) [inline src/app/app.component.ts]: single quotes

**Review by ksenya-ta — CHANGES_REQUESTED** [review]
Good job! Just some minor comments you still need to fix. Also it might be a good idea to write some tests.

**Comment by Olga04051995** [comment]
added code fixes , please take a look

**Review by ksenya-ta — APPROVED** [review]

**Pull request merged**